README.md

```diff
@@ -19,8 +19,9 @@ npm i fastify-leveldb --save
 Add it to you project with `register` and you are done!  
 You can access LevelDB via `fastify.level`.
 ```js
-const fastify = require('fastify')
+const fastify = require('fastify')()
 
+fastify.register(require('fastify-formbody'))
```

Comment:
this should be necessary only if we send a url encoded content type (with application/json it is not necessary).

Maybe to complete the example could you add a `curl` command to let the user try immediately?

Reply:
+1 for adding a curl snippet instead. However this should be covered by some other fastify docs somewhere else.

Reply:
What is the next step? Add a curl example and remove formbody dep? Or simply remove formbody and add curl examples elsewhere? If so, where's the best place to document this?

Reply:
> Or simply remove formbody and add curl examples elsewhere?

I'm +1 on this.
The curl example could be simply at the bottom of the example code.